### PR TITLE
ETH/BTC Conversions

### PR DESCRIPTION
**Pull request by clarencejychan**
# Description
Part of #11 , added in `euro_rate` for BTC/ETH

Fixes: # 11

## Type of change
Enhancement
# Screenshots (if applicable):

### CODE REVIEW

**Comment by clarencejychan** [comment]
Just as a side note, I also noticed that on the dashboard, the graph's always pull the `value` in the `Holdings` scope. I.e., 20,000 CAD would still show up as 20,000 on the graph but the total value would have the converted value. I suggestion might be that each holding always has a `currencyValue` as well which would be updated based on the default currency state and the graphs and displays could use the `currencyValue` .

**Comment by alvesjtiago** [comment]
Thanks @clarencejychan!

So we're trying not to store everything that can be derived from the current values on the database. Since the default currency can be changed at any time, we're trying to not to record the values of each holding for the global currency defined since that can be easily derived on the fly.

Regarding your second question I believe what you're asking is when is `useEffect(() => {}, [])` called right? If that's the case, with the `[]` at the end, that method is called once when the component is mounted. Does that make sense?

File: src/App.js [hunk 1]
```diff
@@ -38,6 +38,22 @@ const updateExchangeRates = () => {
         .write()
     })
   })
+
+  axios.get('https://api.cryptonator.com/api/ticker/eth-eur').then(response => {
```
Comment (alvesjtiago) [inline src/App.js]: Can you change the ticker to `eur-eth`? That way you won't have to divide 1 bu the ticker value 😊

File: src/App.js [hunk 2]
```diff
@@ -38,6 +38,22 @@ const updateExchangeRates = () => {
...
+    const rate = 1 / response.data.ticker.price
+    db.get('currencies')
+      .find({ code: "ETH"})
+      .assign({ euro_rate: rate })
+      .write()
+  })
+  
+  axios.get('https://api.cryptonator.com/api/ticker/btc-eur').then(response => {
```
Comment (alvesjtiago) [inline src/App.js]: Same here, can you change the ticker to eur-btc?

**Comment by clarencejychan** [comment]
Makes sense, I guess we could always do conversions and just create a separate object to put the values so that the graph could use it during some life-cycle hook. 

In regards to the second question, I was more wondering how the `holding` state is even updated from an empty array which it originally is.

**Comment by alvesjtiago** [comment]
Inside the `useEffect` there's a `setHoldings(allHoldings)` which sets the state to the appropriate holdings.

**Comment by clarencejychan** [comment]
OHHHH I see, I re-read the documentation and I missed how setting the state works in functional components. I read `setHoldings` as another variable and not a function. Thanks 👍

**Pull request merged**